webrtc.html

```diff
@@ -7716,18 +7716,6 @@ <h2>Methods</h2>
                   throw <code>InvalidModificationError</code>. This ensures that we always have
                   something to offer, regardless of <code><var>transceiver</var>.<a data-link-for="RTCRTPTransceiver">direction</a></code>.</p>
```

Comment:
This part ensures that we have a "real" codec that we can use for sending, and we have a "real" codec that we can use for receiving. By "real" I mean a codec (e.g. VP8, VP9, H264 or profile thereof), not a line that says we support RTX/RED/FEC.

This means that it will work if direction is sendonly or the direction is recvonly.

What about sendrecv?
Let's assume you have the capability to send VP8 and H264 and receive VP9 and H264. If H264 is not in your list of codecs and the direction is sendrecv, what should be in the SDP? Neither VP8 or VP9 is valid in both directions.